docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+services:
+   app:
+       build:
+           context: .
+       ports:
+           - "8001:8000"

```

Comment:
Port Configuration: The application is exposed on port 8001, which is mapped to the internal port 8000. Ensure this is the intended configuration.